pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -568,4 +568,64 @@ public void testNumGroupsLimit() {
             InstancePlanMakerImplV2.DEFAULT_GROUPBY_TRIM_THRESHOLD));
     assertTrue(brokerResponse.isNumGroupsLimitReached());
   }
+
+  @Test
+  public void testDistinctSum() {
```

Comment:
Please make sure to test both dictionary and no-dictionary scenario

Reply:
As discussed offline, skipping this for now. There are no remaining numeric columns in the avro file that can be used as a noDict column.